discovery/amazon/ec2.go

```diff
@@ -61,24 +61,26 @@ const (
 	ec2LabelSeparator         = ","
 )
 
-// DefaultSDConfig is the default EC2 SD configuration.
-var DefaultSDConfig = SDConfig{
-	Port:            80,
-	RefreshInterval: model.Duration(60 * time.Second),
-}
+var (
+	// EC2DefaultSDConfig is the default EC2 SD configuration.
+	EC2DefaultSDConfig = EC2SDConfig{
```

Comment:
```suggestion
	DefaultEC2SDConfig = EC2SDConfig{
```

Reply:
Updated to `DefaultEC2SDConfig` & `DefaultLightsailSDConfig`